azkaban-common/src/main/java/azkaban/Constants.java

```diff
@@ -31,6 +31,8 @@
  * <p>Job level Properties keys to be put in the {@link JobProperties} class
  */
 public class Constants {
+  // Azkaban Flow Version
+  public static final String AZKABAN_FLOW_VERSION_2_0 = "azkaban.flow.2.0";
```

Comment:
Should this constant name as `AZKABAN_FLOW_VERSION` rather than `AZKABAN_FLOW_VERSION_2_0 `?

Reply:
I'm specifying the flow version here in case we have more versions in the future like AZKABAN_FLOW_VERSION_3_0.